blt/project.yml

```diff
@@ -0,0 +1,53 @@
+project:
```

Comment:
This should be a git mv instead of a total deletion and addition.  This file was removed from / and moved to /blt

By doing a git mv we preserve history.